src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.Metrics.cs

```diff
@@ -45,8 +45,8 @@ partial class DiagController
 
             MetricsPipelineSettings settings = MetricsSettingsFactory.CreateSettings(
                 _counterOptions.CurrentValue,
```

Comment:
So previously we took the counter options and the defaults correct, but not the sdm defualts correct?

Reply:
Yes, expect there are no defaults for SDM; another way to put it, our SDM defaults is to not use SDM.